tests/test_utils_covariance.py

```diff
@@ -160,28 +160,39 @@ def test_covariances_cospectrum():
     assert_array_almost_equal(cosp_pr, cosp_sp, 6)
 
 
-def test_covariances_coherence():
+@pytest.mark.parametrize('coh', ['ordinary', 'instantaneous', 'lagged', 'imaginary', 'truc'])
```

Comment:
```suggestion
@pytest.mark.parametrize('coh', ['ordinary', 'instantaneous', 'lagged', 'imaginary', 'foobar'])
```
more international...